resources/extruders/leapfrog_bolt_pro_extruder_left.def.json

```diff
@@ -0,0 +1,23 @@
+{
+    "id": "leapfrog_bolt_pro_extruder_left",
+    "version": 2,
+    "name": "Left extruder",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "leapfrog_bolt_pro",
+        "position": "1"
```

Comment:
This is going to place the left extruder on the right (and the right extruder on the left)

Reply:
Dear Nallath, Thank you for your feedback.

With the Leapfrog Bolt PRO, T0 is considered right extruder , T1 is considered left extruder. 
We also placed a "1" and "2" above the extruders in the mesh file (leapfrog_bolt_pro_platform.stl) to clarify on this. 

We believe that the naming "Left extruder" or "Right extruder" (for example in slice preview) matches this also.